packages/default/scss/action-sheet/_layout.scss

```diff
@@ -0,0 +1,74 @@
+@include exports("action-sheet/layout") {
+
+    .k-action-sheet-container {
+        width: 100vw;
+        height: 100vh;
+        position: fixed;
+        top: 0;
+        left: 0;
+        overflow: hidden;
+    }
+
+    .k-action-sheet {
+        overflow: auto;
+        position: fixed;
+        padding: 0;
+        left: 0;
+        bottom: 0;
+        z-index: 10002;
+        width: $action-sheet-width;
+        max-width: 100%;
+        max-height: $action-sheet-max-height;
+        transform: translateX(calc(50vw - 50%));
```

Comment:
I believe this could be a variable. 

Also:

1. This is not going to work in IE. (Do we plan to support IE for this component?)
2. `translateX(50vw) translateX(-50%)` will work in IE, and will work faster in all browsers, due to less calculations.